modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/info/item/provider/CalendarBookingInfoItemFormProvider.java

```diff
@@ -64,11 +67,19 @@ private InfoForm _getInfoForm() {
 		return InfoForm.builder(
 		).infoFieldSetEntry(
 			_getBasicInformationInfoFieldSet()
+		).infoFieldSetEntry(
+			_displayPageInfoItemFieldSetProvider.getInfoFieldSet(
+				CalendarBooking.class.getName(), StringPool.BLANK,
+				CalendarBooking.class.getSimpleName(), 0)
```

Comment:
Should the `groupId` be passed as last parameter instead of `0` when the method `public InfoForm getInfoForm(String formVariationKey, long groupId)` is called?

Reply:
Not really; the `itemFormVariationKey` represents things like document types or web content structures (think of "variations" as a kind of subtype). We don't have anything like that for calendar events.